context.go

```diff
@@ -1420,7 +1420,7 @@ func (c *Context) prettyPrintVariant(variations variationMap) string {
 		}
 	}
 
-	return strings.Join(names, ", ")
+	return strings.Join(names, ",")
```

Comment:
With the space removed, it looks like the line wrapping becomes a bit awkward -- why are we removing it?

Reply:
The intention is to have a tighter grouping of variations in a variant than between variants in a list. Iow I think it makes it easier to read

libperfetto_hprof{os:android,image:,arch:arm64_armv8-a,link:shared,version:,art:debug}, libart-disassembler{os:android,image:,arch:arm64_armv8-a,link:shared,version:,art:debug}, libadbconnection{os:android,image:,arch:arm64_armv8-a,link:shared,version:,art:debug}, libart{os:android,image:,arch:arm64_armv8-a,link:shared,version:,art:debug}

than

libperfetto_hprof{os:android, image:, arch:arm64_armv8-a, link:shared, version:, art:debug}, libart-disassembler{os:android, image:, arch:arm64_armv8-a, link:shared, version:, art:debug}, libadbconnection{os:android, image:, arch:arm64_armv8-a, link:shared, version:, art:debug}, libart{os:android, image:, arch:arm64_armv8-a, link:shared, version:, art:debug}

Reply:
So imo this more restrained use of spaces actually improves readability overall, but if you prefer spaces here or in other places just let me know and I'll add them.

Reply:
Thank you. Shall I now submit it myself? I don't find a button for that anywhere.